app/src/main/java/org/mozilla/fenix/compose/list/ExpandableListHeader.kt

```diff
@@ -46,16 +46,21 @@ fun ExpandableListHeader(
     expanded: Boolean? = null,
     expandActionContentDescription: String? = null,
     collapseActionContentDescription: String? = null,
-    onClick: () -> Unit = {},
+    onClick: (() -> Unit)? = null,
     actions: @Composable () -> Unit = {},
 ) {
     Row(
-        modifier = Modifier.fillMaxWidth(),
+        modifier = when (onClick != null) {
+            true -> Modifier.clickable { onClick() }
+            false -> Modifier
+        }.then(
```

Comment:
curious if this section could be wrapped in a `Modifier` extension function

Reply:
Discussed this. This could be a follow up if we feel strongly about it.